js/script.js

```diff
@@ -0,0 +1,6 @@
+const toggleMenu = () => {
+  document.querySelector('.mobile-nav').classList.toggle('block');
+};
```

Comment:
**[OPTIONAL]** I suggest you could try to refactor this code, this will help to make your code look more readable and maintainable. You might find the code referenced below helpful and apply it to other parts of your code as needed.

  ```
const mobileNav  =  document.querySelector('.mobile-nav');
  const toggleMenu = () => {
  mobileNav.classList.toggle('block');
}; 
```